src/utils.ts

```diff
@@ -1,11 +1,22 @@
 import * as vscode from "vscode";
 import * as md5 from "md5";
+import { extname } from "path";
 import { MarkdownNode, Graph } from "./types";
 
 export const findLinks = (ast: MarkdownNode): string[] => {
   if (ast.type === "link" || ast.type === "definition") {
-    return [ast.url!];
+    // ignore empty, anchor and web links
+    if (
+      !ast.url ||
+      ast.url.startsWith("#") ||
+      vscode.Uri.parse(ast.url).scheme.startsWith("http")
```

Comment:
I don't think it is 100% necessary to check for web links as the `filterNonExistingEdges()` util should filter them out, but it doesn't hurt I suppose.